app/model/SubscriptionOps.scala

```diff
@@ -36,6 +45,16 @@ object SubscriptionOps extends LazyLogging {
         nextPlan
       }
     }
+
+    def currentPlans = subscription.plans.list.filter(p => (p.start == now || p.start.isBefore(now)) && p.end.isAfter(now))
```

Comment:
This is what we use to separate current and upcoming plans in /manage. Should we check this in a different way?